src/main/java/org/zalando/nakadi/plugin/api/authz/EventAuthz.java

```diff
@@ -0,0 +1,15 @@
+package org.zalando.nakadi.plugin.api.authz;
+
+public interface EventAuthz
+{
+    public enum CompatibilityMode {
+        FORWARD, COMPATIBLE, NONE;
```

Comment:
If you have to replace all references to these enum values in Nakadi project it might be quite a big change affecting lots of files. Maybe you want to define a new method that "getCompatibilityMode" returning string and then implement it. It would cause a change only in the domain object EventTypeBase and not elsewhere.